### PR TITLE
Clean up add button on reverse inlines

### PR DESCRIPTION
**Pull request by Andrew-Dickinson**
Fix the spacing around the "Add" button that appears under Building inlines, and extend it to Install inlines also

### CODE REVIEW

File: src/meshapi/admin/inlines.py [hunk 1]
```diff
@@ -135,3 +136,20 @@ class InstallInline(BetterInline):
...
+
+class InstallInlineBuildingPage(InstallInline):
+    add_button = True
+    reverse_relation = "building"
+    bulk_add_button = True
+
+
+class InstallInlineMemberPage(InstallInline):
+    add_button = True
+    reverse_relation = "member"
```
Comment (WillNilges) [inline src/meshapi/admin/inlines.py:155]: Do we really need to make a separate class for each of these? Can't we set the `reverse_relation` when we declare/construct it later?

Comment (Andrew-Dickinson) [reply src/meshapi/admin/inlines.py:156]: I don't think so. You can see how we use it below, we just pass a class reference in a list attribute on the `ModelAdmin` class. I can't think of a way to change this property that doesn't pollute the `InstallInline` model (other than the one I've implemented here)

Comment (WillNilges) [reply src/meshapi/admin/inlines.py:156]: Not even if we add a constructor?

**Review by WillNilges — APPROVED** [review]
Don't love having all these separate classes but we've got bigger fish to fry

File: src/meshapi/admin/inlines.py (hunk 1 above)
Comment (Andrew-Dickinson) [reply src/meshapi/admin/inlines.py:156]: Oh no you're totally right. We're passed the model class in literally the first constructor param. Fixed

**Pull request merged**